SetCurrentVersion.cmd

```diff
@@ -1,6 +1,6 @@
 set MAJOR_PREVIOUS=1
 set MINOR_PREVIOUS=5
-set PATCH_PREVIOUS=47
+set PATCH_PREVIOUS=46
```

Comment:
Prev should be 1.6.0, and the new version should be 1.6.1 (really IMO it s.b. 1.7.0 -- this is significant new functionality).